content/implementations.md

```diff
@@ -0,0 +1,88 @@
+---
+doc-note-type: draft
+layout: standalone_resource
+title: "Accessibility Test Tools & Methodologies"
+permalink: /standards-guidelines/act/implementations/
+ref: /standards-guidelines/act/implementations/
+lang: en
+type_of_guidance: false
+footer: ""
```

Comment:
This tries adding "About WCAG 2 Test Rules" and "All WCAG 2 Test Rules" to the header navigation

```suggestion
standalone_resource_header:
  link:
    ref: /standards-guidelines/act/rules/about/
    title: About WCAG 2 Test Rules
standalone_resource_pager:
  icon: @@@ # this is optional
  name: All WCAG 2 Test Rules
  ref: /standards-guidelines/act/rules/
footer: ""
```